labs/b9.md

```diff
@@ -0,0 +1,527 @@
+---
+title: Lab 9 - Version Control and Backups
+layout: lab
+---
+
+## Overview
+
+In this lab, we'll be talking about using version control, in particular Git, for keeping track of changes to your files and backing up your files to other machines. 
+We'll be setting up [Gitolite](http://gitolite.com/gitolite/index.html), an open-source Git server that allows you to host Git repositories on other machines,
+and set access to Git repositories hosted on the Gitolite server.
+
+## A Git Primer
+
+Version control tools are designed to keep track of changes to files, and allow you to rewind to some last saved state. 
+Git, as one of the most popular version control tools in use today, supports these features, but also allows for more advanced workflows such as splitting a filebase into "branches" for experimentation.
+In order to complete this lab, you'll need to know a few basic git commands and some terminology.
+
+### Getting started in git: Initializing/cloning a repository
+
+A "repository" is just a fancy way of saying "a folder where git is set up."
+In order to start working with git, you'll need to either create a repository from scratch,
+or "clone" (download and copy) a repository from somewhere else.
+
+A folder can be made into a git **repository** by running `git init`, telling git that files under the folder can start being tracked by git. When a repository is initialized, git creates a hidden `.git` folder within the root folder to track the files' histories.
+
+```
+bzh@supernova ~> mkdir example-project
+bzh@supernova ~> cd example-project/
+bzh@supernova ~/example-project> git init
```

Comment:
for the sake of not confusing beginner students, I'd s/supernova/tsunami/ and s/~>/$/